projects/core/src/error-handling/multi-error-handler/provide-multi-error-handler.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Provider } from '@angular/core';
+import { LoggingErrorHandler } from './logging-error-handler';
+import { MULTI_ERROR_HANDLER } from './multi-error-handler';
+
+export function provideMultiErrorHandler(): Provider[] {
```

Comment:
[MEDIUM]
Does it really need to be in the public API?

Reply:
Good point, I'll remove it from the `index.ts` file.